thoth/metrics_exporter/jobs/ceph.py

```diff
@@ -35,9 +35,9 @@ class CephMetrics(MetricsBase):
     @register_metric_job
     def get_ceph_connection_error_status() -> None:
         """Check connection to Ceph instance."""
-        inspections = InspectionResultsStore()
+        ceph_storage = ResultStorageBase()
         try:
-            inspections.connect()
+            ceph_storage.connect()
```

Comment:
This will fail on:

https://github.com/thoth-station/storages/blob/c1208e779bd4b9fc284878855a447f03ef937ed0/thoth/storages/result_base.py#L54-L56

As base class `ResultStorageBase` has no `RESULT_TYPE` set.

Reply:
I didn't encounter errors testing `# HELP thoth_ceph_connection_issues Ceph Connection error status. # TYPE thoth_ceph_connection_issues gauge thoth_ceph_connection_issues 0.0`

Reply:
I will correct it, thanks :)